### ISSUE TITLE
Camotoy is working on Christmas

### ISSUE THREAD

**Issue by jkcoxson** [GeyserMC/Geyser#1740]
**Describe the bug**
Camotoy needs to take Christmas off for all the work they've done for this project.

**To Reproduce**
It be December 25

**Expected behavior**
Take a break

**Screenshots / Videos**
![image](https://user-images.githubusercontent.com/28877351/103143955-31602280-46de-11eb-8b73-fe33e3a70f62.png)

**Server Version**
Discord 

**Geyser Version**
Newest, thanks to the hard work here.

**Minecraft: Bedrock Edition Version**
My iPhone 

**Additional Context**
Thanks for all the work you do, we appreciate it!😀

**Comment by lukeeey** [comment]
it’s okay, camotoy got a nice christmas present :)

**Comment by Camotoy** [comment]
Hi. I appreciate all the love. :) 

Issues are really easy for me to manage, don't worry. I have been taking a break from the Discord support channels though! xd I've had a wonderful day outside of my life here on GitHub and Discord.

Coding is my passion these days - I love developing for Geyser and seeing it grow, and the awesome community. I'm so grateful that I found this project with a great leader (Redned), wonderful sidekick (Tim), a frienemy (at first - Luke), and a fellow developer who developed alongside me (RTM). I love seeing contributors put input into the project. I'm so incredibly grateful to our helpers and everyone else who bears support from the masses. 

I think we've got a great 2021 incoming, and I'm not planning on going anywhere.

Thanks for all the love and support. :)
